.circleci/config.yml

```diff
@@ -202,7 +202,7 @@ workflows:
           docker-username: DOCKER_HUB_USER
           image: 'prefecthq/server'
           path: '.'
-          extra_build_args: '--build-arg PREFECT_VERSION=master --build-arg RELEASE_TIMESTAMP=$(date -u +"%Y-%m-%dT%H:%M:%SZ")'
+          extra_build_args: '--build-arg PREFECT_VERSION=master --build-arg BUILD_DATE=$(date -u +"%Y-%m-%dT%H:%M:%SZ")'
```

Comment:
`RELEASE_TIMESTAMP` is a setting that is actually used within the application, so we need to keep it